dot/utils_test.go

```diff
@@ -33,7 +33,7 @@ func TestCreateJSONRawFile(t *testing.T) {
 				bs: &BuildSpec{genesis: NewTestGenesis(t)},
 				fp: filepath.Join(t.TempDir(), "test.json"),
 			},
-			expectedHash: "f7f1b82c0ba16b20e36bfb462d7899af2c76728918f639f5c5ef0e91ff3e7077",
+			expectedHash: "ddedbc626806eb0f2a2eec5ade59a7141fedbff55e27021e3265a5f9000dec2e",
```

Comment:
Why is this different?

Reply:
[One of the commits ](https://github.com/ChainSafe/gossamer/commit/acf0d94bc141d6ee1c4a4a89da6bb30d57195548) actually changed this after @kishansagathiya rebased this branch previously.  I wasn't sure why initially, but given that @dimartiro moved in the in memory trie in `development`, I got the previous hash when building the test trie.